### PR TITLE
Minor updates to 3D preprocessing layers

### PR DESCRIPTION
**Pull request by lengzq**
1. frustum_random_dropping_points.py: Change the random_point_mask shape to [1, points, 1] to align with the global drop point logic.

2. swap_background.py: Convert the masks used in boolean_mask from float tensors to bool tensors.

### CODE REVIEW

File: keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def get_random_transformation(self, point_clouds, **kwargs):
         frustum_mask = tf.concat(frustum_mask, axis=0)
         random_point_mask = (
             self._random_generator.random_uniform(
-                [num_frames, num_points, 1], minval=0.0, maxval=1
+                [1, num_points, 1], minval=0.0, maxval=1
```
Comment (tanzhenyu) [inline keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:105]: any reason for this? Seems we're making it less generic

Comment (tanzhenyu) [reply keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:105]: If we need to make this, can you add a comment to why this is the case

Comment (lengzq) [reply keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:105]: Similar to global drop point. We use the same point mask for all frames.

**Review by ianstenbit — APPROVED** [review]
Could you please update the PR description to something like 

"Minor updates to 3D preprocessing layers"

Thanks!

File: keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py (hunk 1 above)
Comment (lengzq) [reply keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:106]: Done.

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**